python/src/aiconfig/editor/server/server.py

```diff
@@ -132,15 +132,19 @@ def save() -> FlaskResponse:
     state = get_server_state(app)
     aiconfig = state.aiconfig
     request_json = request.get_json()
+    path: str | None = request_json.get("path", None)
 
-    _op = make_op_run_method(MethodName("save"))
+    if path is None:
+        if aiconfig is None:
+            return HttpResponseWithAIConfig(message="No AIConfig loaded", code=400, aiconfig=None).to_flask_format()
```

Comment:
Shouldn't we do this check if there is a path, too?

Reply:
It is already being done in `run_aiconfig_operation_with_op_args`. Should we do it again here explicitly?

Reply:
Right, yes, no need to duplicate.